app/models/hubstats/pull_request.rb

```diff
@@ -58,10 +64,31 @@ def self.create_or_update(github_pull)
           end
       end
 
+      if user.team && user.team.id
+        pull_data[:team_id] = user.team.id
+      end
+
       return pull if pull.update_attributes(pull_data)
       Rails.logger.warn pull.errors.inspect
     end
 
+    # Public - Updates the team_id for all pull requests from the past year
+    #
+    # Returns - nothing
+    def self.update_teams_in_pulls
+      puts "Gathering pulls from past year"
+      pulls = created_in_past_year # only update team_ids from PRs that were created in the past year, uncomment below line if wish to update all PRs
+      # pulls = Hubstats::PullRequest.all
+      pulls.each do |pull|
+        user = Hubstats::User.where(id: pull.user_id).first
```

Comment:
You could turn the body of this iteration block into an instance method on pull_request and such that it becomes:

pulls.each { |pull| pull.assign_team_from_user }